### PR TITLE
Swapfile role

### PR DESCRIPTION
**Pull request by tomca32**
Deploy was failing due to not having enough memory to compile nokogiri, so this adds a swapfile ansible role that creates a swap file of a size of 2GB.

### CODE REVIEW

**Comment by tomca32** [comment]
@j16r :bowtie:

**Comment by j16r** [comment]
I'd vendor in the role.

**Comment by tomca32** [comment]
Not sure what you mean. The role is installed by your ansible to wherever you install ansible roles. The only thing here is the roles.txt file. I don't think you mean to put that in vendor, since that's a file we edit.

**Comment by j16r** [comment]
I usually just make the role a submodule or subtree rather than ask people
to install it, is more reproducable.

**Comment by tomca32** [comment]
All right, that makes sense.

**Comment by tomca32** [comment]
Added to deploy/roles

File: deploy/requirements.yml [hunk 1]
```diff
@@ -2,4 +2,4 @@
 # Install roles with
```
Comment (j16r) [inline deploy/requirements.yml]: Is this file needed anymore?

Comment (tomca32) [inline deploy/requirements.yml]: Not necessarily, but I would keep it so we can easily update roles (although there's only one now).

Comment (j16r) [inline deploy/requirements.yml]: I think we should vendor and maintain all our roles internally to keep consistency, it'll mean that anyone can pull down our repo and it should work. Otherwise you tend to get role drift (ansible has no way to pick role versions). Keeps things simple too.

**Comment by tomca32** [comment]
@j16r - fair points. Removed requirements.yml

**Comment by j16r** [comment]
:shipit: though Ideally I'd like to somehow take 'compilation' off the server, this is a good fix!

**Comment by tomca32** [comment]
Yeah I agree, does ruby or bundler have any good mechanisms to ship a gem bundle?
